torch_geometric/sampler/__init__.py

```diff
@@ -0,0 +1,5 @@
+from torch_geometric.sampler.neighbor_sampler import NeighborSampler
+
+__all__ = [
+    'NeighborSampler',
```

Comment:
We can consider to also expose `utils`.

Reply:
With the simplification of utils as a result of removing `edge_type_to_str` and `edge_type_from_str`, etc, I am not sure this is necessary at the current moment; perhaps in the future.